review/models.py

```diff
@@ -14,16 +12,8 @@ class Rating(models.TextChoices):
 
 
 class Review(models.Model):
-    description = models.TextField(blank=True)
     rating = models.CharField(max_length=2, choices=Rating.choices, default=Rating.UNSPECIFIED)
```

Comment:
Are there any methods? 
Like number of reviews ? average score from maximum

Reply:
I updated the code and the description of the PR. I also used shell to ensure the functionality works.